src/components/PerformanceRanking/PerformanceRanking.tsx

```
@@ -50,7 +50,8 @@ export interface PerformanceRankingData {
 /**
  * Props for the PerformanceRanking component
  */
-export interface PerformanceRankingProps extends HTMLAttributes<HTMLDivElement> {
+export interface PerformanceRankingProps
+  extends HTMLAttributes<HTMLDivElement> {
```

ESLint

[error] Delete `⏎·`

(prettier/prettier)